app.js

```diff
@@ -1,34 +1,62 @@
-// import functions and grab DOM elements
+import { getRandomThrow } from './get-random-throw.js';
+import { checkResults } from './get-random-throw.js';
 
-const button = document.querySelector('#guess-button');
-
-const results = document.querySelector('#displayed-results');
+const guessButton = document.querySelector('#guess-button');
+// const resetButton = document.querySelector('#reset-button');
+//const radio = document.querySelector('input:checked');
+const numOfGames = document.querySelector('#num-of-games');
+const numOfTies = document.querySelector('#num-of-ties');
+const numOfWins = document.querySelector('#num-of-wins');
+const numOfLosses = document.querySelector('#num-of-losses');
 
 // initialize state
-let winsData = 0;
 let totalGuesses = 0;
+let gamesWon = 0;
+let gamesLost = 0;
+let gameTies = 0;
 
 // set event listeners to update state and DOM
-button.addEventListener('click', () => {
-    
+guessButton.addEventListener('click', () => {
     totalGuesses++;
 
-    const checkedRadio = document.querySelector('input:checked');
-    const userGuess = checkedRadio.value;
-
-    let correctGuess = null;
-
-    const myNumber = Math.random();
-
-
-    // generate a random correct answer: heads or tails
-    
+    numOfGames.textContent = totalGuesses;
+
+    const radios = document.querySelector('input:checked');
+    let userGuess = radios.value;
+
+    let computerChoice = getRandomThrow();
+    checkResults(userGuess, computerChoice);
+
+    if (checkResults(userGuess, computerChoice) === 'WIN!') {
+        gamesWon++;
+        numOfWins.textContent = gamesWon;
+    } else if (
+        checkResults(userGuess, computerChoice) === 'LOSE') {
+        gamesLost++;
+        numOfLosses.textContent = gamesLost;
+    } else {
+        gameTies++;
+        numOfTies.textContent = gameTies;
+    }
+    // console.log(gamesWon, 'games won');
+    // console.log(gameTies, 'game ties');
+    // console.log(gamesLost, 'games lost');
+    // console.log(totalGuesses, 'total guesses');
+
+    numOfWins.textContent = `gamesWon=${gamesWon}`;
+    numOfTies.textContent = `gameTies=${gameTies}`;
+    numOfLosses.textContent = `gamesLost=${gamesLost}`;
+    numOfGames.textContent = `totalGuesses=${totalGuesses}`;
+});
+
+// resetButton.addEventListener('click', () => {
+//     totalGuesses = 0;
+//     gamesWon = 0;
+//     gameTies = 0;
+//     gamesLost = 0;
     
-    // Math.random(Math.random() * 3);
-
-    // we need to compar the user guess to the correct answer
-    totalGuesses.textContent = "total: " + totalGuesses;
-    wins.textContent = "total: " + totalGuesses;
-    losses.textContent = "total: " + totalGuesses;
-    current.textContent = "total: " + lossesData;
-});
\ No newline at end of file
+//     numOfWins.textContent = gamesWon;
+//     numOfTies.textContent = numOfTies;
+//     numOfLosses.textContent = numOfLosses;
+//     numOfGames.textContent = totalGuesses;
+// });
```

Comment:
files should end with a new line. You can configure VS code to automatically add a new line at the end of each file: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files